specify/specify7
# Add a tool in record set dialog to merge RS together

Fixes #1665

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

- open record set side bar menu 
- click button merge 
- select min 2 RS 
- [ ] verify you can only select RS that are from the same table 
- click merge 
- wait for the page to reload 
- [ ] verify in the record set side bar menu that the RS have been merged, a new one has been created and the merged ones have been deleted. The new RS has the name of the first or one of the selected RS.
